### PR TITLE
add prince pack one

### PR DESCRIPTION
**Pull request by Peter-Egham**

### CODE REVIEW

File: server/src/cardCollection/faction.ts
Comment (Robsmash138) [inline server/src/cardCollection/faction.ts:1]: you have "doug-st.john" replace the . with a dash; the naming convention only has dashes and no other special characters

Comment (Robsmash138) [reply server/src/cardCollection/faction.ts:1]: Also in disciplines you have them all quoted; these are not quoted in this case and look like this { animalism:1 } or { [discipline]:[integer] } except blood sorcery which will be "blood sorcery":[integer]

Comment (Robsmash138) [reply server/src/cardCollection/faction.ts:1]: In my card (Robert Bruce) it mentions: "If this is the only character you control". should say "If this is the only character in your coterie"

Comment (Robsmash138) [reply server/src/cardCollection/faction.ts:1]: "When this character mends 1 [blood], put 1 additional [blood] from your general supply on them." in Krisken Monroes text should be: "When this character mends, put 1 additional [blood] from your general supply on them."

Comment (Robsmash138) [reply server/src/cardCollection/faction.ts:1]: In Doug St. John's name field- add a space between St. and John for searching purposes

**Review by Robsmash138 — CHANGES_REQUESTED** [review]
see comments on faction.ts in the commits tab

File: server/src/cardCollection/faction.ts
Comment (stopfstedt) [reply server/src/cardCollection/faction.ts:1]: all handed. :+1:

**Review by stopfstedt — CHANGES_REQUESTED** [review]
Drakh's images are missing, please add `pp1-drakh.png` and `pp1-drakh.webp` to `/assets/cards`.

Otherwise, this is good to go.

**Review by stopfstedt — APPROVED** [review]
LGTM

**Review by Robsmash138 — APPROVED** [review]
looks good!

**Pull request merged**